source/support/decoding-metadata.rst

```diff
@@ -0,0 +1,80 @@
+=================
+Decoding Metadata
+=================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+
+Metadata
+--------
+
+MinIO stores object metadata for erasure coded objects in ``xl.meta`` files.
+
+Each disk in the set containing the object has this file.
+The file is a binary format, requiring a tool to view the values.
+
+You can download the metadata for one or more objects with the MinIO Console's Support > Inspect tool or using :mc-cmd:`mc support inspect` at the command line.
+
+
+.. _minio-xl-meta-decode-tool:
+
+xl-meta Decode Tool
+-------------------
+
+MinIO provides xl-meta tool to decode the contents of an ``xl.meta`` file to a JSON format.
+
+Install xl-meta Decode Tool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The xl.meta Decode Tool requites Go. To install Go, `download <https://golang.org/dl/>`_ the package for your operating system.
+
+With Go installed, run the following command:
+
+.. code-block:: shell
+   :class: copyable
+
+   go install github.com/minio/minio/docs/debugging/xl-meta@latest
+
+
+Using xl-meta
+-------------
+
+Locate ``xl.meta`` Files and Decode to JSON
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Executing ``xl-meta`` looks for an ``xl.meta`` file in the current folder.
+If found, **xl-meta** decodes the file to JSON. 
+
+.. code-block:: shell
+   :class: copyable
+
+   xl-meta
```

Comment:
Maybe we can include an example using `mc support inspect play/file` and then `xl-meta file/xl.meta` or whatever the output would be.